src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/AuthorizationEnabledIntegrationTest.java

```diff
@@ -154,9 +158,15 @@
             "User status must be Active to perform this operation";
     protected static final String ACCESS_IS_DENIED_ERROR_MESSAGE = "Access is denied";
 
+    protected static final String FORBIDDEN_LD = "Forbidden with Launch Darkly";
+
```

Comment:
is this string used anywhere?

Reply:
This is used in tests returns_launchDarkly_forbidden_when_retrieve_organisations_info_with_invalid_flag.